Library/Homebrew/formula.rb

```diff
@@ -261,6 +261,15 @@ def installed?
     (dir = installed_prefix).directory? && dir.children.length > 0
   end
 
+  # If this {Formula} has been modified by the user.  Relies on git.
+  def adulterated?
```

Comment:
Let's add `return false unless which("git")` to handle the case where git is unavailable. This can happen for Linuxbrew.